src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -160,6 +160,13 @@ Here is the basic process you would use to obfuscate the sensitive data in this
     Obfuscation expressions **can** be reusable: they are totally agnostic to how the log attribute containing the sensitive data is named. For instance, the Social Security expression defined above could be applied to a log attribute named `ssn`, `socialSecurityNumber`, or `socSecNum`.
 
     But still, you can create non-reusable obfuscation expressions (like the `Born date (loginService specific)` one) that are tightly coupled to the log attribute's format. For example, you could use whatever comes after `born on` and `before`.
+
+    <Callout variant="caution" title="Nested regular expression capture groups not supported">
+    Main responsability of this feature is to replace sensitive data with the hash or mask in an easy and performant way.
+    If you want or need to group different matches inside a capture group, remember to use the non capture group syntax `(?:{EXPRESSION})` to avoid creating nested capture groups.
```

Comment:
```suggestion
    If you want or need to group different matches inside a capture group, remember to use the non-capture group syntax `(?:{EXPRESSION})` to avoid creating nested capture groups.
```